cardano-db-sync/src/Cardano/DbSync/Api.hs

```diff
@@ -187,8 +197,23 @@ generateNewEpochEvents env details = do
         , esEpochNo = Strict.Just currentEpochNo
         }
 
+getTopLevelConfig :: SyncEnv -> TopLevelConfig CardanoBlock
+getTopLevelConfig syncEnv =
+  case envLedgerEnv syncEnv of
+    HasLedger hasLedgerEnv -> Consensus.pInfoConfig $ leProtocolInfo hasLedgerEnv
+    NoLedger noLedgerEnv -> Consensus.pInfoConfig $ nleProtocolInfo noLedgerEnv
```

Comment:
I'm unsure about this one as it's used in `runSyncNodeClient` when creating `codecConfig` so had to add it `NoLedgerEnv` 🤷